src/main/java/org/example/movieSpring/controller/MovieController.java

```diff
@@ -0,0 +1,51 @@
+package org.example.movieSpring.controller;
+
+import org.example.movieSpring.model.Movie;
+import org.example.movieSpring.service.MovieService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequestMapping(path = "api/v1")
+public class MovieController {
+
+    private final MovieService movieService;
+
+    @Autowired
```

Comment:
the annotation should be on the parameter but can be omited in the constructor

Reply:
Does this mean moving the @Autowired to 

public MovieService(@Autowired MovieRepo movieRepo) {
    this.movieRepo = movieRepo;
}

Reply:
nvm fixed

Reply:
> Does this mean moving the @Autowired to
> 
> public MovieService(@Autowired MovieRepo movieRepo) { this.movieRepo = movieRepo; } ?

yupp, and if you only have 1 constructor you can omit it completely

Reply:
would love to have a white board sitdown on why that is early next week, like why its needed when we have multiple constructors but not 1